frappe/core/page/data_import_tool/data_import_tool.js

```diff
@@ -42,11 +42,15 @@ frappe.DataImportTool = Class.extend({
 				if(me.doctype) {
 
 					// render select columns
-					var doctype_list = [frappe.get_doc('DocType', me.doctype)];
+					var parent_doctype = frappe.get_doc('DocType', me.doctype);
+					parent_doctype["req"] = "True";
+					var doctype_list = [parent_doctype];
+					
 					frappe.meta.get_table_fields(me.doctype).forEach(function(df) {
-						doctype_list.push(frappe.get_doc('DocType', df.options));
+						var d = frappe.get_doc('DocType', df.options);
+						df.reqd&&(d.req="True");
```

Comment:
`d.req = df.reqd`

also why different fieldnames `req` and `reqd` ? Keep the fieldname as `reqd` everywhere